docs/administering/ssl.md

```diff
@@ -21,10 +21,9 @@ You have a few options.
 
 To share port 443 with other services on the same machine:
 
-- You [can install `sniproxy` to share port
-  443](https://xamar.sandcats.io/shared/Bqa9dftNbc1Ni06D-SgBdkFuM_iky8VHAlTw0Rk1lzN) between your
-  existing server and Sandstorm so that Sandstorm can manage (and autorenew) its own certificates.
-  This allows you to combine an **existing web server on port 443** with free sandcats.io HTTPS.
+- You [can install `sniproxy` to share port 443](sniproxy.md) between your existing server and Sandstorm so that
+  Sandstorm can manage (and autorenew) its own certificates. This allows you to combine an **existing
+  web server on port 443** with free sandcats.io HTTPS.
   
 - You [can follow this guide](https://juanjoalvarez.net/es/detail/2017/jan/12/how-set-sandstorm-behind-reverse-proxy-keeping-you/)
```

Comment:
This site appears to be down. Don't know whether that's permanent or transient. Not really related to this pr but I noticed while reviewing.

Reply:
I'll look into it. Good catch.

Reply:
I'll swap it out for the archive.org link for now, and then revisit the entire topic (and the worthiness of the linked page) in my next PR.